starter-code/css/main.css

```diff
@@ -0,0 +1,75 @@
+h1 a{
+	color: #ffb782;
+	letter-spacing: 2px;
+	font-weight: 500;
+	text-decoration: none;
+	font-size: 48px;
+	
+}
+
+body {
+	font-family: Lato;
+	padding: 30px 925px 0px 50px;

```

Comment:
This is a pretty huge right padding value. Did you mean to add that much space on the entire body?